client/my-sites/google-my-business/stats/chart.js

```diff
@@ -113,10 +134,11 @@ export default connect(
 		return {
 			siteId,
 			interval,
-			data: FakeData.statFunction( ownProps.statType, interval ),
+			data: getGoolgeMyBusinessSiteStats( state, siteId, ownProps.statType, interval, 'total' ),
```

Comment:
This will spit out a bunch of errors on the console when it is null because the underlying `PieChart` is expecting it. Probably out of scope for this PR, just a note.